docs/citing.rst

```diff
@@ -6,35 +6,110 @@
+  </p>
+  <p>
+  4. 	Tustison NJ, Avants BB, Cook PA, Zheng Y, Egan A, Yushkevich PA, Gee JC. N4ITK: improved N3 bias correction. IEEE Trans Med Imaging [Internet]. 2010 Jun;29(6):1310–20. Available from: <a href="http://dx.doi.org/10.1109/TMI.2010.2046908">http://dx.doi.org/10.1109/TMI.2010.2046908</a>
+  </p>
+  <p>
+  5. 	Dale A, Fischl B, Sereno MI. Cortical Surface-Based Analysis: I. Segmentation and Surface Reconstruction. Neuroimage. 1999;9(2):179–94.
+  </p>
+  <p>
+  6. 	Fonov VS, Evans AC, McKinstry RC, Almli CR, Collins DL. Unbiased nonlinear average age-appropriate brain templates from birth to adulthood. NeuroImage; Amsterdam [Internet]. 2009 Jul 1;47:S102. Available from: <a href="https://search.proquest.com/docview/1833173019">https://search.proquest.com/docview/1833173019</a>
+  </p>
+  <p>
+  7. 	Avants BB, Epstein CL, Grossman M, Gee JC. Symmetric diffeomorphic image registration with cross-correlation: evaluating automated labeling of elderly and neurodegenerative brain. Med Image Anal [Internet]. 2008 Feb;12(1):26–41. Available from: <a href="http://dx.doi.org/10.1016/j.media.2007.06.004">http://dx.doi.org/10.1016/j.media.2007.06.004</a>
+  </p>
+  <p>
+  8. 	Jenkinson M, Bannister P, Brady M, Smith S. Improved optimization for the robust and accurate linear registration and motion correction of brain images. Neuroimage [Internet]. 2002 Oct;17(2):825–41. Available from: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12377157">https://www.ncbi.nlm.nih.gov/pubmed/12377157</a>
+  </p>
+  <p>
+  9. 	Andersson JLR, Skare S, Ashburner J. How to correct susceptibility distortions in spin-echo echo-planar images: application to diffusion tensor imaging. Neuroimage [Internet]. 2003 Oct;20(2):870–88. Available from: <a href="http://dx.doi.org/10.1016/S1053-8119(03)00336-7">http://dx.doi.org/10.1016/S1053-8119(03)00336-7</a>
+  </p>
+  <p>
+  10. 	Cox RW. AFNI: software for analysis and visualization of functional magnetic resonance neuroimages. Comput Biomed Res [Internet]. 1996 Jun;29(3):162–73. Available from: <a href="http://www.ncbi.nlm.nih.gov/pubmed/8812068">http://www.ncbi.nlm.nih.gov/pubmed/8812068</a>
+  </p>
+  <p>
+  11. 	Jenkinson M. Fast, automated, N-dimensional phase-unwrapping algorithm. Magn Reson Med [Internet]. 2003 Jan;49(1):193–7. Available from: <a href="http://dx.doi.org/10.1002/mrm.10354">http://dx.doi.org/10.1002/mrm.10354</a>
+  </p>
+  <p>
+  12. 	Huntenburg JM. Evaluating nonlinear coregistration of BOLD EPI and T1w images [Internet]. Freie Universität Berlin; 2014. Available from: <a href="http://pubman.mpdl.mpg.de/pubman/item/escidoc:2327525/component/escidoc:2327523/master_thesis_huntenburg_4686947.pdf">http://pubman.mpdl.mpg.de/pubman/item/escidoc:2327525/component/escidoc:2327523/master_thesis_huntenburg_4686947.pdf</a>
```

Comment:
This appears to be the preferred link (one click away from the actual PDF): http://hdl.handle.net/11858/00-001M-0000-002B-1CB5-A